deploy/start.sh

```diff
@@ -4,6 +4,12 @@ export ASSISTED_SERVICE_URL="${ASSISTED_SERVICE_URL:-http://localhost:8090}"
 
 envsubst '$ASSISTED_SERVICE_URL' < /deploy/nginx.conf > "$NGINX_DEFAULT_CONF_PATH/nginx.conf"
 
+if [ "$ASSISTED_SERVICE_SCHEME" = "https" ]; then
+    envsubst 'HTTPS_CERT_FILE' < /deploy/nginx_ssl.conf > "/deploy/nginx_ssl.conf.tmp"
+    envsubst 'HTTPS_KEY_FILE'  < /deploy/nginx_ssl.conf.tmp > "$NGINX_DEFAULT_CONF_PATH/nginx_ssl.conf"
```

Comment:
```suggestion
    envsubst 'HTTPS_KEY_FILE'  < /deploy/nginx_ssl.conf.tmp > "$NGINX_DEFAULT_CONF_PATH/nginx_ssl.conf"
```
How does ngnix know it should use this file instead of `nginx.conf`. We'll end up having both files inside `$NGINX_DEFAULT_CONF_PATH/` when `"$ASSISTED_SERVICE_SCHEME" = "https"`

Reply:
See this `nginx.conf` (grabbed it from the UI container): https://gist.github.com/nmagnezi/58a7e6e2f87fc1ff820aaea656738927#file-nginx-conf-L65-L66

Lines 65-66:
```
#        # Load configuration files for the default server block.
#        include /opt/app-root/etc/nginx.default.d/*.conf;
```